Loenn/entities/zipline.lua

```diff
@@ -0,0 +1,15 @@
+local zipline = {}
+
+zipline.name = "isaBag/zipline"
+zipline.depth = 0
+zipline.texture = "isafriend/objects/zipline/handle"
+zipline.nodeTexture = "isafriend/objects/zipline/handle_end"
+zipline.nodeLimits = {0, -1}
+zipline.nodeVisibility = "always"
+zipline.nodeLineRenderType = "fan"
```

Comment:
not sure what these choices are for nodes. use `{0, 2}` for limits, and it should be rendering the actual line like in the ahorn plugin